Plugins/LoggerPlugin/LoggerPlugin.swift

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import PackagePlugin
+import Foundation
+
+@main
+struct LoggerPlugin: BuildToolPlugin {
+  func createBuildCommands(context: PluginContext, target: Target) async throws -> [Command] {
+    let outputDirectory = context.pluginWorkDirectory.appending("GeneratedSources")
+    guard let target = target as? SourceModuleTarget else {
+      Diagnostics.error("Attempted to use `LoggerPlugin` on an unsupported module target")
+      return []
+    }
+    
+    try FileManager.default.createDirectory(atPath: outputDirectory.string, withIntermediateDirectories: true)
+    let source = """
+    import Foundation
+    import os.log
+    
+    let log = Logger(subsystem: "com.brave.ios", category: "\(target.moduleName.lowercased())")
```

Comment:
I did not find a way to not hardcode the prod bundle id. Was thinking maybe it could be dynamic, but the plugin does not see Bundle.main.bundleID